new/LIT6624Maqabis.xml

```diff
@@ -0,0 +1,113 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6624Maqabis" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ዜና፡ ሕይወቱ፡ ዘብፁዕ፡ መቃቢስ፡ ዘተጋደለ፡ በገዳመ፡ ሆር፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Zenā ḥǝywatu za-bǝḍuʿ Maqābis za-tagādala ba-gadāma Hor</title>
+                <title xml:lang="en" corresp="#t1">Story of the life of the blessed Maqābis who struggled in the desert of Hor</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Ascetic teachings and episodes from the life of Maqābis of Hor.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="MonasticLiterature"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2022-05-02">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT6624Maqabis" passive="BLorient770"/>
+                    <relation name="lawd:hasAttestation" active="LIT6624Maqabis" passive="BNFabb59"/>
```

Comment:
I think we need the person record and the relation to the person

Reply:
I can create him of course, but his identity remains quite unclear (I assume a desert father).

Reply:
If he is the protagnist, do we need a personal record?

Reply:
> I can create him of course, but his identity remains quite unclear (I assume a desert father).

Well we do know that he is venerated in the Synaxarion, this is justification enough

Reply:
https://github.com/BetaMasaheft/Persons/pull/556